api/src/main/java/org/apache/cloudstack/api/command/user/vm/StartVMCmd.java

```diff
@@ -82,6 +82,13 @@ public class StartVMCmd extends BaseAsyncCmd implements UserCmd {
                since = "3.0.1")
     private Long hostId;
 
+    @Parameter(name = ApiConstants.CONSIDER_LAST_HOST,
+            type = CommandType.BOOLEAN,
+            description = "True by default, CloudStack will firstly try to start the VM on the last host where it run on before stopping, if destination host is not specified. " +
+                    "If false, CloudStack will not consider the last host and start the VM by normal process.",
+            since = "4.18.0")
+    private Boolean considerLastHost;
```

Comment:
The default value should be false, to maintain current behavior.

Reply:
@JoaoJandre 
if you have looked into the code, you may find that the last host is considered as the first option when start a vm.
the current behavior is same as considerLastHost=true.

Reply:
Ok, I analyzed the code and what you're saying makes sense.

Reply:
thanks @JoaoJandre for reviews